.requirements

```diff
@@ -10,5 +10,5 @@ RESTY_LMDB_VERSION=1.0.0
 RESTY_EVENTS_VERSION=0.1.3
 ATC_ROUTER_VERSION=1.0.0
 LIBYAML_VERSION=0.2.5
-KONG_BUILD_TOOLS_VERSION=4.33.11
+KONG_BUILD_TOOLS_VERSION=chore/test_3_0_release
```

Comment:
This needs to be changed before merge.

Reply:
.. or so I thought :) - I'll downgrade to 4.33.16 of kbt so that we can merge this.